docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Database version information
+{: .no_toc }
+
+{% include /cloud-db/cloud-summary-db-version.md %}
+
+{% include page-toc.md %}
+
+## Before you begin
+* [Learn about Cloud databases](/docs/cloud/cloud-databases/cloud-db-manage)
+* [Create a FeatureBase database](/docs/cloud/cloud-databases/cloud-db-create)
+
+{: .note}
+A database that fails to respond will display their version as "Unknown".\
+[Learn more about unknown database versions](/docs/cloud/cloud-troubleshooting/issue-cloud-version-unknown/)
+
+## Find database version in list of databases
+1. Select databases on the sidebar
+2. View your database's version under the version column
+
+## Find database version in database summary
+1. Select databases on the sidebar
+2. Click on your database's name
+3. View your database's version in the summary
+
+## Further Information
+Further information about a database's version can be found [here](/docs/cloud/cloud-troubleshooting/issue-cloud-version-unknown/).
```

Comment:
Nope! Nope! Nope! There is  no "one-word-hyperlink-anchor-text" anymore.

Take a beak at https://github.com/FeatureBaseDB/featurebase-docs/blob/main/help-on-help/writing-help/writing-hyperlinks.md

Links under "Before you begin" and "Further information" should be single bullets and the anchor text explains what they are.

So in this case:

```
## Further information
* [Learn about database version issues](/docs/cloud/cloud-troubleshooting/issue-cloud-version-unknown)
```
And the trailing slash will probably break the link FYI

You can run broken link checks using these instructions:

https://github.com/FeatureBaseDB/featurebase-docs/blob/main/help-on-help/publishing-workflow/broken-link-check.md